usr.sbin/adduser/adduser.8

```diff
@@ -52,6 +53,8 @@ utility is a shell script, implemented around the
 command, for adding new users.
 It creates passwd/group entries, a home directory,
 copies dotfiles and sends the new user a welcome message.
+It also supports creating a ZFS dataset for the user's
```

Comment:
```
On systems where the parent of home directory is a dataset,
.Nm
will create the home directory as a ZFS dataset by default,
unless the system administrator specified otherwise.
```

Reply:
Thanks for the detailed review, @delphij.  My thought was to allow the administrator to specify both the zfs dataset and home directory separately and map the home directory onto the dataset by setting the mountpoint.  What you're saying is to simplify it by creating a zfs dataset only if the homeprefix sits on an existing dataset and provide an opt-out flag if the admin doesn't want zfs datasets?